Formula/netdata.rb

```diff
@@ -1,8 +1,8 @@
 class Netdata < Formula
   desc "Diagnose infrastructure problems with metrics, visualizations & alarms"
   homepage "https://netdata.cloud/"
-  url "https://github.com/netdata/netdata/releases/download/v1.23.2/netdata-v1.23.2.tar.gz"
-  sha256 "761aec15901e09f963361752c6e5b5cb723e342a3c5bf6b0624067fceb2ccdfd"
+  url "https://github.com/netdata/netdata/releases/download/v1.24.0/netdata-v1.24.0.tar.gz"
+  sha256 "e7d6691fdce893b5a341fc2ee11b3b4f951e7b426031fae4178e669066c264c0"
   license "GPL-3.0"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

ref: https://github.com/netdata/netdata#license

Note that `netdata` redistributes software with different licenses (see https://github.com/netdata/netdata/blob/master/REDISTRIBUTED.md) that may need to be added in a comment for future adoption.

It looks like these are: `MIT`, `BSD-3-Clause`, `BSD-2-Clause`, `CC-BY-4.0 `, `LGPL-2.1-only`, `Apache-2.0`, `WTFPL`, `Unlicense`

Reply:
All the licenses are permissive licenses, bundled with `GPL-3.0-or-later`, it fits into the relicensing story that I mentioned before.

![image](https://user-images.githubusercontent.com/1580956/89965424-c7f1e280-dc1a-11ea-935d-fe881a932526.png)

Reply:
note, `GPLv3` is a strong copyleft license